DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
+#define DataFormats_METReco_HcalPhase1FlagLabels_h
+
+#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
+
+namespace HcalPhase1FlagLabels
+{
+    enum HBHEStatusFlag {
+        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
+        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
+        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
+        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
+        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
```

Comment:
The flags were discussed with the Hcal Noise Group.
All flags that are currently being used for noise cleaning
are preserved.

I always had an impression that JME is just using
HcalNoiseSummary, and the code making that
object is maintained by the Hcal Noise Group.

On 08/04/2016 07:48 PM, Slava Krutelyov wrote:
>
> In DataFormats/METReco/interface/HcalPhase1FlagLabels.h <https://github.com/cms-sw/cmssw/pull/15324#discussion_r73627401>:
>
> > @@ -0,0 +1,32 @@
> > +#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
> > +#define DataFormats_METReco_HcalPhase1FlagLabels_h
> > +
> > +#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
> > +
> > +namespace HcalPhase1FlagLabels
> > +{
> > +    enum HBHEStatusFlag {
> > +        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
> > +        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
> > +        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
> > +        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
> > +        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
>
> was this flag duality clearly agreed in HCAL DPG and JME POG? (JME is the primary customer of the flags). Now it looks like you are creating a parallel universe of flags for the phase-1 hits.
>
> @mariadalfonso <https://github.com/mariadalfonso> @mmarionncern <https://github.com/mmarionncern> @schoef <https://github.com/schoef>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15324/files/f2734c7b29b226480f1efc887de88c8ae73f121f#r73627401>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS4-SO0ydDyCp9GLnbGlsWw6vjq_3i3ks5qcoh0gaJpZM4JXio9>.
>